connector/queue/model.py

```diff
@@ -213,6 +213,43 @@ def autovacuum(self):
         jobs.unlink()
         return True
 
+    @api.model
+    def requeue_stuck_jobs(self, in_queue_delta=5, started_delta=180):
```

Comment:
```suggestion
    def requeue_stuck_jobs(self, enqueued_delta=5, started_delta=0):
```
`enqueued_delta` is better than `in_queue_delta` as it reflects the name of the state.
We don't know if users in the wild have "legit" jobs that runs more than 3 hours, so we should probably not set any default value for `started_delta`. A value of 5 minutes for `enqueued_delta` is fine as it should not happen.

Update the docstring to say that 0 means it is not checked